src/Mod/Draft/DraftSnap.py

```diff
@@ -97,9 +97,30 @@ def __init__(self):
         self.callbackMove = None
         self.snapObjectIndex = 0
 
+        # snap keys, it's important tha they are in this order for
+        # saving in preferences and for properly restore the toolbar
+        self.snaps = ['Lock',           # 0 
+                      'Near',           # 1 used by Near Snap
+                      'Extension',      # 2
+                      'Parallel',       # 3
+                      'Grid',           # 4
+                      "Endpoint",       # 5
+                      'Midpoint',       # 6
+                      'Perpendicular',  # 7
+                      'Angle',          # 8
+                      'Center',         # 9
+                      'Ortho',          # 10
+                      'Intersection',   # 11
+                      'Special',        # 12
+                      'Dimensions',     # 13 TODO change to lowercase
+                      'WorkingPlane'    # 14 TODO change to lowercase
```

Comment:
If you are going to test strings, you can handle this by transforming the string to lowercase, and testing this.

For example,
```py
"Dimensions".lower() == "dimensions"

"DimEnsIoNS".lower() == "dimensions"
```

Reply:
i forgot to change the comments, at the end I had to match exactly the same name of Draft Snap Gui Tools to automatize several things, so this can't be turned lowercase animore...